packages/story-editor/src/components/keyboardShortcutsMenu/landmarkShortcuts.js

```diff
@@ -53,10 +54,10 @@ function LandmarkShortcuts() {
     <LandmarksWrapper>
       {landmarks.map(({ label, shortcut }) => (
         <Landmark key={label}>
+          <ShortcutLabel keys={shortcut} />
```

Comment:
This change renders the `<dt>` element _after_ the `<dd>` element. I'm not sure how strict the standard is about their order in the markup (if `<dt>` must absolutely come before `<dd>`). This change was necessary to allow correct text selection from top-left to bottom-right.